package.json

```diff
@@ -14,30 +14,67 @@
     "build": "tsc && vite build",
     "bundle-types": "api-extractor run",
     "postbuild": "npm run bundle-types && rm -rf ./dist/.temp",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "prettier": "prettier --write 'src/**/**/*.{js,ts,tsx,scss}'",
+    "prettier:check": "prettier --check 'src/**/**/*.{js,ts,tsx,scss}'",
+    "lint": "eslint 'src/**/**/*.{js,ts,tsx}'",
+    "tsc": "tsc --noEmit",
+    "storybook": "start-storybook -s ./src/assets -p 6006",
+    "build-storybook": "build-storybook"
   },
   "license": "MIT",
   "peerDependencies": {
     "baseui": ">=12.2.0",
-    "react": ">=17.0.2",
-    "react-dom": ">=17.0.2"
+    "react": ">= 17.0.2 < 19",
+    "react-dom": ">= 17.0.2 < 19",
+    "styletron-react": ">=6.1.0 < 7"
```

Comment:
We don't  need to provide styletron(becuase engine is optional thing in baseweb) in this package. Actually it should be dev dependency for storybook but nothing else.